### ISSUE TITLE
UnsupportedOperationException - WindowsFileSystemProvider for takari-jar (release-1.10)

### ISSUE THREAD

**Issue by javapapo** [takari/takari-lifecycle#5]
Multi module, maven project, using the takari-maven-plugin. When we upgraded yesterday to version 1.10 in one of our takari-jar modules, (most probably the first one to be packaged by the reactor) the following exception happened.

Windows 7, Sun Java JDK 7, MVN 3.2.3

```java
INFO] ------------------------------------------------------------------------
ERROR] Failed to execute goal io.takari.maven.plugins:takari-lifecycle-plugin:1.10.0:jar (default-jar) on project eBanking-message-resources: Execution default-jar of goal io.takari.m
rg.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal io.takari.maven.plugins:takari-lifecycle-plugin:1.10.0:jar (default-jar) on project eBanking-message-reso
       at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:224)
       at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:153)
       at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:145)
       at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:116)
       at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:80)
       at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build(SingleThreadedBuilder.java:51)
       at org.apache.maven.lifecycle.internal.LifecycleStarter.execute(LifecycleStarter.java:120)
       at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:347)
       at org.apache.maven.DefaultMaven.execute(DefaultMaven.java:154)
       at org.apache.maven.cli.MavenCli.execute(MavenCli.java:582)
       at org.apache.maven.cli.MavenCli.doMain(MavenCli.java:214)
       at org.apache.maven.cli.MavenCli.main(MavenCli.java:158)
       at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
       at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
       at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
       at java.lang.reflect.Method.invoke(Method.java:606)
       at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced(Launcher.java:289)
       at org.codehaus.plexus.classworlds.launcher.Launcher.launch(Launcher.java:229)
       at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode(Launcher.java:415)
       at org.codehaus.plexus.classworlds.launcher.Launcher.main(Launcher.java:356)
aused by: org.apache.maven.plugin.PluginExecutionException: Execution default-jar of goal io.takari.maven.plugins:takari-lifecycle-plugin:1.10.0:jar failed.
       at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:143)
       at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:208)
       ... 19 more
aused by: java.lang.UnsupportedOperationException
       at sun.nio.fs.WindowsFileSystemProvider.readAttributes(WindowsFileSystemProvider.java:192)
       at java.nio.file.Files.readAttributes(Files.java:1686)
       at java.nio.file.Files.getPosixFilePermissions(Files.java:1953)
       at io.tesla.proviso.archive.FileMode.getFileMode(FileMode.java:280)
       at io.tesla.proviso.archive.source.FileEntry.getFileMode(FileEntry.java:49)
       at io.tesla.proviso.archive.ArchiveHandlerSupport.createEntryFor(ArchiveHandlerSupport.java:12)
       at io.tesla.proviso.archive.Archiver.archive(Archiver.java:116)
       at io.takari.maven.plugins.jar.Jar.archive(Jar.java:160)
       at io.takari.maven.plugins.jar.Jar.access$6(Jar.java:156)
       at io.takari.maven.plugins.jar.Jar$1.create(Jar.java:99)
       at io.takari.incrementalbuild.aggregator.internal.DefaultAggregatorBuildContext$DefaultAggregateOutput.createIfNecessary(DefaultAggregatorBuildContext.java:120)
       at io.takari.maven.plugins.jar.Jar.executeMojo(Jar.java:87)
       at io.takari.maven.plugins.TakariLifecycleMojo.execute(TakariLifecycleMojo.java:99)
       at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:132)
       ... 20 more
ERROR]
ERROR]
ERROR] For more information about the errors and possible solutions, please read the following articles:
ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginExecutionException
ERROR]
ERROR] After correcting the problems, you can resume the build with the command
ERROR]   mvn <goals> -rf :eBanking-message-resources
ress any key to continue . . .
```

**Comment by mosabua** [comment]
Seems to be related to Windows as we get related problems in the usage of the Android Maven Plugin as well. 

https://github.com/simpligility/android-maven-plugin/pull/564#issuecomment-70239404

**Comment by ifedorenko** [comment]
should be fixed in 1.10.1, but give it an hour or two to appear in central.

**Comment by mosabua** [comment]
Its there
